pkg/image/api/validation/validation.go

```diff
@@ -10,13 +11,21 @@ import (
 	"github.com/openshift/origin/pkg/image/api"
 )
 
+func MinimalNameValidation(name string, prefix bool) (bool, string) {
```

Comment:
> It'd be nice to have a single common func that all validations could use to check the bare minimum

My first attempt as rejected because it was too broad.  My second attempt failed because our main validation package would become cyclic.  I decided I'd rather get enough in to stop us from failing before attempting a more consistent change again.